docs/profiling.md

```diff
@@ -0,0 +1,121 @@
+# vHive profiling tool
+
+The tool focus on the performance characteristics of vHive framework. It uses a practical method, 
+[the TopDown method](https://ieeexplore.ieee.org/document/6844459), to identify potential 
+performance bottlenecks by colocating VMs on a single host. The tool issues requests to VMs in
+Round-Robin and takes metrics from the VMs, such as RPS per CPU, hardware counters, etc. If multiple
+groups of metrics are collected, then line charts can be drawn for each attribute of metrics. An example 
+is shown in the [section](###TestProfileIncrementConfiguration-function) below.
+
+## Methodology
+
+The tool includes the following components:
+- ***A loader function*** issues requests to vHive step by step until the RPS reaches the maximum. 
+  At each step, the function issues a percentage of the maximum RPS (e.g., 5% of the maximum RPS) 
+  and profile counters.
+- ***A latency measurement goroutine*** measures the latency by injecting requests in 
+  Round-Robin and measures runtimes of requests to compute the mean latency and the 
+  90-percentile latency.
+- ***A bind function*** binds VMs in two ways:
+  -  If profile CPU ID is set, the tool allocates only one VM to the physical core where the CPU is
```

Comment:
I don't understand this sentence at all. You should first say that the tool collects counters from a single core then explain why your tool binds a VM to that core, not vice versa.

Reply:
Done